indico/modules/events/registration/__init__.py

```diff
@@ -106,7 +107,9 @@ def _visible_registration(event):
                 .has_rows())
 
     def _visible_participant_list(event):
-        return event.has_feature('registration')
+        if not event.has_feature('registration'):
+            return False
+        return not any(values_from_signal(signals.event.hide_participant_list.send(sender, event=event)))
```

Comment:
The sender is unused (I believe it's None), so I would not pass it on to this signal. What about using the event itself as the sender for `hide_participant_list` instead?